st2mistral/actions/stackstorm.py

```diff
@@ -80,7 +80,7 @@ def run(self):
             )
         )
 
-        endpoint = self.st2_context['api_url'] + '/actionexecutions'
+        endpoint = self.st2_context['api_url'] + '/executions'
```

Comment:
We renamed the endpoint ages ago and it requires some hacking to make it still work in st2api v2.

I'll check itests on this repo, but if it's something required for PR to be merged, it should probably be added to circle.yml. The rest of the tests work.